packages/phone-number-privacy/combiner/src/match-making/get-contact-matches.ts

```diff
@@ -4,15 +4,29 @@ import { authenticateUser, isVerified } from '../common/identity'
 import {
   hasValidAccountParam,
   hasValidContractPhoneNumbersParam,
+  hasValidPhoneNumberHash,
   hasValidUserPhoneNumberParam,
-  isBodyReasonablySized,
 } from '../common/input-validation'
 import logger from '../common/logger'
 import { getDidMatchmaking, setDidMatchmaking } from '../database/wrappers/account'
 import { getNumberPairContacts, setNumberPairContacts } from '../database/wrappers/number-pairs'
 
+interface GetContactMatchesRequest {
+  account: string
+  userPhoneNumber: string
+  contactPhoneNumbers: string[]
+  hashedPhoneNumber: string
+}
+
+interface ContactMatch {
```

Comment:
just noting that you have a `ContactMatch` interface in the mobile code with different fields - confused me for a sec when searching

Reply:
Renamed the mobile one